tests/tests/optionsTests.robot

```diff
@@ -71,15 +71,42 @@ Find and open Extraction language filter
 Find and open Collision resolution option
     [Documentation]    Click Option, then Collision resolution, select every option and verify if Reload Badge appears/disappears
     [Tags]    options
-
     Click Element    ${OptionsButton}
     Wait Until Element Is Visible    ${OptionsList}
     Select From List By Index    collisionResolutionOptions    0
     List Selection Should Be    collisionResolutionOptions    overwrite
+    Wait Until Element Is Not Visible    ${ReloadBadge}
     Select From List By Index    collisionResolutionOptions    1
     List Selection Should Be    collisionResolutionOptions    rename
+    Wait Until Element Is Visible    ${ReloadBadge}
     Select From List By Index    collisionResolutionOptions    2
     List Selection Should Be    collisionResolutionOptions    rename-all
+    Wait Until Element Is Visible    ${ReloadBadge}
     Select From List By Index    collisionResolutionOptions    3
     List Selection Should Be    collisionResolutionOptions    error
+    Wait Until Element Is Visible    ${ReloadBadge}
+
+Exclude temporary files functionality test
+    [Documentation]    Exclude temporary files functionality removes tmp folder from loaded file
+    [Tags]    options
+
+    ${downloaded_file_path}    Set Variable    ${DOWNLOAD_PATH}${test_setup}[archive_name].zip
 
+    Click Add Files Button
+    Ubuntu Upload Test File    ${test_setup}[path]
```

Comment:
changed in "tests" to "Upload Test File"

Reply:
Done